packages/@aws-cdk/aws-sagemaker/lib/model-data.ts

```diff
@@ -0,0 +1,104 @@
+import * as crypto from 'crypto';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as assets from '@aws-cdk/aws-s3-assets';
+import { Construct } from 'constructs';
+import { IModel } from './model';
+
+// The only supported extension for local asset model data
+const ARTIFACT_EXTENSION = '.tar.gz';
```

Comment:
do you think this might be extended by sagemaker anytime soon? if there's a chance that they do, it feels more maintainable to have this be a list:

```ts
const ARTIFACT_EXTENSIONS = ['.tar.gz'];
```